projects/kit/pipes/filter-by-input/filter-by-input.pipe.ts

```diff
@@ -25,6 +25,7 @@ export class TuiFilterByInputPipe implements PipeTransform {
     private readonly textfield = inject(TuiTextfieldComponent, {optional: true});
     private readonly host = inject(TUI_DATA_LIST_HOST);
 
+    public transform<T>(items: readonly T[], matcher?: TuiStringMatcher<T>): readonly T[];
     public transform<T>(items: T, matcher?: TuiStringMatcher<TuiArrayElement<T>>): T;
```

Comment:
Do we really need the second one with `TuiArrayElement`? Does anything break if you only leave the new overload?

Reply:
fixed